core/services/keystore/keys/ocr2key/key_bundle.go

```diff
@@ -106,7 +106,7 @@ func (raw Raw) Key() (kb KeyBundle) {
 	case chaintype.StarkNet:
 		kb = newKeyBundle(new(starknet.OCR2Key))
 	default:
-		panic(chaintype.NewErrInvalidChainType(temp.ChainType))
+		return nil
```

Comment:
Should log this?  From the way we treat these keys, they sound very much like candidates for pruning out of the DB.

Reply:
Going to push this now because it's blocking people, I will add logging in another PR